posthog/src/main/java/com/posthog/PostHogConfig.kt

```diff
@@ -103,6 +103,11 @@ public open class PostHogConfig(
      * The hook is called before the event is cached or sent over the wire
      */
     public var propertiesSanitizer: PostHogPropertiesSanitizer? = null,
+    /**
+     * Hook that allows to sanitize the event
+     * The hook is called before the event is cached or sent over the wire
+     */
+    public var beforeSendBlock: Array<(PostHogEvent) -> PostHogEvent?> = emptyArray(),
```

Comment:
```suggestion
    public var beforeSend: Array<(PostHogEvent) -> PostHogEvent?> = emptyArray(),
```

I think we can make it a list as well using mutableListOf